docs/getting-started/features.md

```diff
@@ -2,206 +2,230 @@
 
-## Port Forwarding (Windows)
+</TabItem>
+</Tabs>
 
-To forward a port:
+## WSL (Windows)
 
-1. Find the service and click **Forward**. A random port is assigned.
-1. Optional: click **Cancel** to remove the port assigned.
+The option to make the Rancher Desktop Kubernetes configuration accessible to any Linux distributions configured for WSL. Once enabled, you can communicate with the Rancher Desktop Kubernetes cluster using tools like `kubectl` from within the WSL distribution.
 
-## Supporting Utilities (macOS & Linux)
+With WSL, memory and CPU allocation is configured globally across all Linux distributions. Refer to the [WSL documentation] for instructions.
 
-A Rancher Desktop installation comes with command-line utilities that are used to interface with its various features. Some examples of these utilities are `docker`, `nerdctl`, `kubectl`, and `helm`. These utilities are located in `~/.rd/bin` - you may see which utilities are included in your installation by running `ls ~/.rd/bin`.
+[WSL documentation]:
+https://docs.microsoft.com/en-us/windows/wsl/wsl-config#options-for-wslconfig
 
-In order to use the utilities, `~/.rd/bin` must be in the `PATH` variable in your shell. On the **Application Settings** tab there are two options for doing this: **Automatic** or **Manual**. **Automatic** `PATH` management will add `~/.rd/bin` to your `PATH` by modifying your shell .rc files for you. **Manual** `PATH` management does not change anything - in this mode you must add `~/.rd/bin` to your `PATH` yourself.
+## Virtual Machine (macOS & Linux)
 
-## Images
+### Memory
 
-The **Images** tab, allows you to manage the images on your virtual machine.
+The amount of memory to allocate to Rancher Desktop. The selectable range is based on your system. The red area within the range indicates an allocation that may affect system services.
 
-To manage your images using nerdctl instead, refer to the [Images](../tutorials/working-with-images.md) section.
+### CPUs
 
-### Scanning Images
+The number of CPUs to allocate to Rancher Desktop. The selectable range is based on your system. The red area within the range indicates an allocation that may affect system services.
 
-This feature uses [Trivy] to scan your images for vulnerabilities and configuration issues.
+## Container Runtime
 
-To scan an image:
+Set the [container runtime] for Rancher Desktop. Users have the option of [containerd] which provides namespaces for containers and the use of nerdctl or [dockerd (moby)] which enables the Docker API and the use of the Docker CLI. Only one container runtime will function at a time.
 
-1. From the image list, find the image you want to scan.
-1. Click **⋮ > Scan**.
-1. Review the results then click **Close Output to Continue**.
+When switching to a different container runtime:
 
-[Trivy]:
-https://github.com/aquasecurity/trivy
+- A restart of Kubernetes is required.
```

Comment:
I'm not sure if we really need to warn of Kubernetes restarts like this anymore. I guess it's technically true, it's just that the restart is automatically triggered on apply.

Reply:
Feedback applied.